plugins/kubernetes/app/models/kubernetes/build_job_executor.rb

```diff
@@ -91,6 +101,31 @@ def fill_job_details(k8s_job, build, project, tag:, push: false, tag_as_latest:
       k8s_job[:spec][:template][:spec][:containers][0].update(container_params)
     end
 
+    # Stub this, otherwise hard to test
```

Comment:
set the CLAIR_EXEC_SCRIPT to a tempfile you create, then put either `ls` or bogus in there to test it